Guide/src/reference/openvmm/management.md

```diff
@@ -0,0 +1,7 @@
+# Configuration and Management
+As [acknowledged in the disclaimer](../../user_guide/openvmm.md#notable-features) management interfaces in OpenVMM are unstable and incomplete. You can do basic VM operations such as stop, restart, save,  restore, pause, resume, etc.. via the interactive console. You can deploy a VM with various configurations such as number of processors, RAM size, UEFI, graphic console, etc.. and devices such as virtual NIC, vTPM, etc.. via the CLI. gRPC / ttrpc support functionality similar to the interactive console.
```

Comment:
Please try to keep markdown wrapped at 80 columns.

(I know we're somewhat inconsistent on this, but its what we typically aim for)